package.json

```diff
@@ -35,14 +34,14 @@
   "dependencies": {
     "@buf/connectrpc_eliza.bufbuild_es": "1.10.0-20230913231627-233fca715f49.1",
     "@buf/connectrpc_eliza.connectrpc_es": "1.4.0-20230913231627-233fca715f49.3",
-    "@buf/cosmos_ibc.bufbuild_es": "1.10.0-20240703151202-d7191877c701.1",
-    "@buf/cosmos_ibc.connectrpc_es": "1.4.0-20240703151202-d7191877c701.3",
-    "@buf/penumbra-zone_penumbra.bufbuild_es": "1.10.0-20240703080008-312294d02bf9.1",
-    "@buf/penumbra-zone_penumbra.connectrpc_es": "1.4.0-20240703080008-312294d02bf9.3",
+    "@buf/cosmos_ibc.bufbuild_es": "1.10.0-20230913112312-7ab44ae956a0.1",
+    "@buf/cosmos_ibc.connectrpc_es": "1.4.0-20230913112312-7ab44ae956a0.3",
```

Comment:
these two moved backwards, because i noticed it's what core uses https://github.com/penumbra-zone/penumbra/blob/main/proto/buf.lock